maven-cli/src/test/java/org/apache/maven/cling/invoker/forked/DefaultForkedMavenInvokerTest.java

```diff
@@ -20,28 +20,34 @@
 
 import java.nio.file.Files;
 import java.nio.file.Path;
+import java.nio.file.Paths;
 
+import org.apache.maven.api.cli.ParserRequest;
 import org.apache.maven.cling.invoker.ProtoLogger;
 import org.apache.maven.cling.invoker.mvn.forked.DefaultForkedMavenInvoker;
 import org.apache.maven.cling.invoker.mvn.forked.DefaultForkedMavenParser;
 import org.apache.maven.jline.JLineMessageBuilderFactory;
-import org.junit.jupiter.api.Disabled;
+import org.junit.jupiter.api.Assumptions;
 import org.junit.jupiter.api.Test;
 import org.junit.jupiter.api.io.TempDir;
 
-@Disabled("Works ONLY with Maven4!")
 public class DefaultForkedMavenInvokerTest {
     @Test
     void smoke(@TempDir Path tempDir) throws Exception {
+        // works only in recent Maven4
+        Assumptions.assumeTrue(Files.isRegularFile(
+                Paths.get(System.getProperty("maven.home")).resolve("conf").resolve("maven.properties")));
```

Comment:
Does this mean they should be ITs ?

Reply:
Um, no, they just require this system property to be set (surefire in POM does it) AND that conf/maven.propeties exists that was introduced on latest Maven4